docs/Configuration.md

```diff
@@ -4,18 +4,105 @@ title: Configuration
 permalink: /configuration/
 nav_order: 4
 ---
-# Configuration 
+
+# Configuration
 
 VS Code settings enable you to customize various aspects of its function. The InterSystems extensions provide settings used to configure VS Code for ObjectScript development.
 
+{: #code-configuration-basic}
+
+## Basic Configuration
+
+VS Code has a concept of a workspace, which is a set of directories you want to use when you're working on a particular project. In the simplest setup when you are working within a single directory, a VS Code workspace is just the root folder of your project. In this case you keep your settings in two files inside a `.vscode` directory located at the root of your project. Those two files are `settings.json`, which contains most configuration settings, and `launch.json`, which contains debugging configurations.
+
+Here is the simplest `settings.json` file contents:
+
+{: #code-workspace-simple}
+
+```json
+{
+    "objectscript.conn": {
+        "ns": "USER",
+        "active": true, 
+        "host": "localhost", 
+        "port": 52773, 
+        "username": "_SYSTEM" 
+    }
+}
+```
+
+However, a better strategy is to let the InterSystems Server manager handle the server connection information as described [later](#config-server). That extension also allows you to store your password securely, so please use it. Then in `settings.json` you only need to specify the server name:
+
+```json
+{
+    "objectscript.conn": {
+        "server": "iris",
+        "ns": "USER",
+        "active": true
+    }
+}
+```
+
+If you need ObjectScript compilation flags other than the default, add an `"objectscript.compileFlags"` property to `settings.json` (more compileFlags options are [covered here](/vscode-objectscript/settings/)):
+
+{: #code-workspace-compileFlags}
+
+```json
+{
+    "objectscript.conn": {
+        "server": "iris",
+        "ns": "USER",
+        "active": true, 
+    },
+    "objectscript.compileFlags": "cuk/compileembedded=1"
+}
+```
+
+Here is the simplest `launch.json` file contents, which debugs the method `Test` in the class `Example.Service`, passing 2 parameters as input (see ["Running and Debugging"](/vscode-objectscript/rundebug/) for more information):
+
+{: #code-workspace-simple-debug}
+
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "objectscript",
+            "request": "launch",
+            "name": "Example.Service.Test", 
+            "program": "##class(Example.Service).Test(<methodparam1>,<methodparam2>)"
```

Comment:
Do we support <...> placeholders like this? If not I suggest you use some actual values so as to illustrate how doublequotes have to be escaped in JSON, e.g.

```suggestion
            "program": "##class(Example.Service).Test(\"answer\",42)"
```

(please test the above)